docs/book/v3/features/container/config.md

```diff
@@ -255,5 +255,5 @@ container configuration.
 Examples of how the above capabilities may be implemented include:
 
 - [zendframework/zend-auradi-config](https://github.com/zendframework/zend-auradi-config)
-- [zendframework/zend-pimple-config](https://github.com/zendframework/zend-auradi-config)
-- [jsoumelidis/zend-sf-di-config](https://github.com/zendframework/zend-auradi-config)
+- [zendframework/zend-pimple-config](https://github.com/zendframework/zend-pimple-config)
+- [jsoumelidis/zend-sf-di-config](https://github.com/zendframework/jsoumelidis/zend-sf-di-config)
```

Comment:
It should be `https://github.com/jsoumelidis/zend-sf-di-config` (remove `zendframework/` part in the link).